infrastructure/kubernetes/monitoring/ingress.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: monitoring
+  namespace: monitoring
+  annotations:
+    # kubernetes.io/ingress.class: "gce"
+    # # The static IP has to be created with the following command:
+    # # `gcloud compute addresses create keep-monitoring-ingress --global`
+    # kubernetes.io/ingress.global-static-ip-name: "keep-monitoring-ingress"
+    # networking.gke.io/managed-certificates: monitoring-cert
```

Comment:
We need to create an external IP and SSL cert for the monitoring hub (both using terraform, see example for `mezo-rcp.yaml`) and attach it here:
```
  annotations:
    kubernetes.io/ingress.global-static-ip-name: mezo-staging-monitoring-hub-external-ip
    ingress.gcp.kubernetes.io/pre-shared-cert: mezo-staging-monitoring-hub-ssl-certificate
```

Note this is different from the IP we added https://github.com/mezo-org/mezod/pull/434 that will be used by the `metrics-scraper`. Just realized that we probably could name it differently but too late, let's stick with what we have.

Of course, this must be also parametrized through Kustomize as we need different names for testnet and mainnet.

Reply:
yep.